server/rpc/interceptor/auth/auth_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"testing"
```

Comment:
I realize that there's no need to create a new mockQueryClient struct as it already exists in `mocks/query_client.go.` 
Therefore, I will delete this unnecessary struct soon."